lib/models/users.js

```diff
@@ -13,6 +14,9 @@ async function setUser(data) {
 async function getUser({ username }) {
   return await collection("users").findOne({ username: username });
 }
+async function getUserImage(username) {
+  return await collection("users").findOne({ username: username });
+}
```

Comment:
Looks more or less the same than the getUser function. Maybe you can just use one function for both.

Reply:
i tried but then i got an error, because in one i deconstruct the user and in the other not. Happy for suggestions :)

Reply:
Not sure but maybe you could destructure the user in the [function call](https://github.com/kschmitz123/diy-project-app/blob/52a2a0670b6b82d7ca89a105a2f291cdb8d5dcf8/lib/routes/users.js#L10)

Maybe something like this: `const {user} = await getUser(user);`